code/def_files/main-f.sdr

```diff
@@ -291,7 +291,7 @@ void main()
    // Normal map - convert from DXT5nm
    vec2 normalSample;
    normal.rg = normalSample = (texture(sNormalmap, texCoord).ag * 2.0) - 1.0;
-   normal.b = sqrt(1.0 - dot(normal.rg, normal.rg));
```

Comment:
After conversation with Kobrar and DahBlount on Discord, the old line was the correct behavior and the fix should probably be just to clamp it to avoid values of 0 (clamping to between, say, 0.0001 and 1.0 should be safe enough).